web_widget_one2many_product_picker/README.rst

```diff
@@ -244,6 +247,8 @@ Contributors
     * Pedro M. Baeza
     * David Vidal
 
+* Giovanni Serra <giovanni@gslab.it>
```

Comment:
Why this addition? In case of any contribution of this guy, it should be in a separate commit for respecting attribution